firefox-ios/firefox-ios-tests/Tests/SyncIntegrationTests/Pipfile.lock

```diff
@@ -542,11 +535,11 @@
         },
         "pytest-fxa-mte": {
             "hashes": [
-                "sha256:5feb85e691b48ca10da27ab31e5ebd19ed38eda4fa03b1ca3b467bf49af188f3",
-                "sha256:6d8d7d1f840963e2b5589e0be7bf2c3ba080d1ce70549677879e24c0268c772c"
+                "sha256:c66be9d51b32414cf5cc6b84dfe5bd2ae468d9ba301a3199306a17de70e8f1a7",
+                "sha256:e10b9b6038725f600f381447246e3d844ee806fd4277f9307bd08b70d5ba29c9"
             ],
             "index": "pypi",
-            "version": "==1.5.1"
+            "version": "==1.6.0"
```

Comment:
The library is now hosted in PyPI: https://pypi.org/project/pytest-fxa-mte/#history